tethys_gizmos/static/tethys_gizmos/css/tethys_map_view.min.css

```diff
@@ -1 +1,244 @@
\ No newline at end of file
+/*****************************************************************************
```

Comment:
this doesn't look minified.

Reply:
@sdc50 tethys_map_view.min.css is now compressed.